enterprise/frontend/src/embedding-sdk/components/public/MetabaseProvider.tsx

```diff
@@ -48,6 +55,7 @@ export const MetabaseProviderInternal = ({
   className,
 }: InternalMetabaseProviderProps): JSX.Element => {
   const { fontFamily = DEFAULT_FONT } = theme ?? {};
+  useInitData({ config });
```

Comment:
Is `useInitData` not already added to some other component?

Reply:
It was used here: https://github.com/metabase/metabase/pull/47764/files#diff-5c5c38f17eca5fe21b81ce25c487a3ab8f587a3200da195887158d7b9fe6397bL1
But we don't need that component anymore